incubator/mysql/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+MySQL can be accessed via port 3306 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To connect to your database run the following command:
+
+   kubectl run {{ template "fullname" . }}-client --rm --tty -i --image bitnami/mysql --command -- mysql -h {{ template "fullname" . }} {{- if .Values.mysqlRootPassword }} -p {{ .Values.mysqlRootPassword }}{{- end -}}
```

Comment:
this command does not work because of the space character between the flag and value of `-u` and `-p` arguments, it should be `kubectl run {{ template "fullname" . }}-client --rm --tty -i --image bitnami/mysql --command -- mysql -h{{ template "fullname" . }} {{- if .Values.mysqlRootPassword }} -p{{ .Values.mysqlRootPassword }}{{- end -}}`